app/models/api_key.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+# The API key model
+class ApiKey < Sequel::Model
+  DIGEST = 'SHA256'
+  SALT = 'Ontohub API Key'
```

Comment:
I think the standard way is to generate a new salt for each key and save it alongside the hash in the database. However, since we only allow unique hashes, this might not be a problem (I think).

Reply:
I adapted this from Devise, where the salt is only `"Devise #{column_name}"`, like `"Devise password"`.  I agree with you that generating a salt for each row is the usual way. I can change this, if you want me to. It's not a big deal.

Reply:
I think it's better to change it now than to have to change it later

Reply:
Wait... it's not possible for API keys. For a user, it's no problem because we sign in with a user ID and therefore, we can lookup the salt in the database. We don't have an API key ID and hence, we cannot lookup the salt in the database during key verification.

Reply:
True, in that case :+1: